Prac05/birth_date_dict.py

```diff
@@ -0,0 +1,24 @@
+"""
+Program asks user for the date-of-birth details for 5 people, and displays their individual ages by using a dictionary.
+"""
+
+#TODO Finish this!!!!
+
+birth_date_dict = {}
```

Comment:
this would be probably better in the main since over here it is a global variable